### DISCUSSION TITLE
Calculation of Fg, Fh and fraction transported

### DISCUSSION THREAD

**Discussion by DeniTue** [Open-Systems-Pharmacology/Forum#1470] [Questions & Problems]
Dear OSP-Community,

Does anyone have experience with calculating Fg (fraction escaping gut wall metabolism) and Fh (fraction excaping hepatic metabolism) in PK-Sim? In addition, is it possible for transporter substrates to display the fraction transported?

Many thanks for you help.

Best,
Denise

**Comment by AndreDlm** [comment]
Hi Denise, a simple workaround to estimate the "first-pass fg" has been discussed here: https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/517

André

**Comment by Yuri05** [comment]
>In addition, is it possible for transporter substrates to display the fraction transported?

See the discussion in https://github.com/Open-Systems-Pharmacology/Forum/discussions/1463 on how to estimate the "Amount transported". Then you just need to divide by the applied drug mass.

**Comment by DeniTue** [comment]
Thank you for your suggestions.

Best,
Denise
